schemas/com.snowplowanalytics.snowplow.storage/snowflake_config/jsonschema/2-0-0

```diff
@@ -0,0 +1,120 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Snowplow Snowflake storage configuration",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.storage",
+        "name": "snowflake_config",
+        "format": "jsonschema",
+        "version": "2-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        },
+        "auth": {
+            "description": "Authentication method for Snowflake Load",
+            "oneOf": [
+                {
+                    "type": "object",
+                    "properties": {
+                        "accessKeyId": {
+                            "type": "string"
+                        },
+                        "secretAccessKey": {
+                            "type": "string"
+                        }
+                    },
+                    "required": ["accessKeyId", "secretAccessKey"]
+                },
+                {
+                    "type": "object",
+                    "properties": {
+                        "roleArn": {
+                            "type": ["string", "null"],
+                            "minLength": 20
+                        },
+                        "sessionDuration": {
+                            "type": "integer",
+                            "minimum": 900
+                        }
+                    },
+                    "required": ["roleArn", "sessionDuration"]
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        },
+        "awsRegion": {
+            "type": "string"
+        },
+        "manifest": {
+            "type": "string"
+        },
+        "snowflakeRegion": {
+            "type": "string"
+        },
+        "database": {
+            "type": "string"
+        },
+        "account": {
+            "type": "string"
+        },
+        "warehouse": {
+            "type": "string"
+        },
+        "schema": {
+            "type": "string"
+        },
+        "input": {
+            "type": "string",
+            "format": "uri"
+        },
+        "stage": {
+            "type": "string"
+        },
+        "stageUrl": {
+            "type": "string",
+            "format": "uri"
+        },
+        "badOutputUrl": {
```

Comment:
I think this should be required. If we're going to make the Transformer work on with FS, do you think it would validate `file:///some/path`-like URIs?

Reply:
Yeah, I think so. I will try to look at the FS support in the following days whether adding it would be easy or not. I can be more precise about this after looking at it.